.rubocop.yml

```diff
@@ -13,41 +13,118 @@ Bundler/OrderedGems:
   Enabled: false
 Gemspec/OrderedDependencies:
   Enabled: false
+Layout/ClosingParenthesisIndentation:
+  Enabled: false
+Layout/FirstArrayElementIndentation:
+  EnforcedStyle: consistent
 Layout/LineLength:
+  Max: 100
+  Exclude:
+    - "spec/**/*_spec.rb"
+Layout/SpaceInLambdaLiteral:
   Enabled: false
-Lint/StructNewOverride:
+Layout/MultilineMethodCallIndentation:
+  Enabled: true
+  EnforcedStyle: indented
+Layout/SpaceInsideHashLiteralBraces:
   Enabled: true
+  EnforcedStyle: no_space
+  EnforcedStyleForEmptyBraces: no_space
+Lint/BooleanSymbol:
+  Enabled: false
+Lint/StructNewOverride:
+  Enabled: false
```

Comment:
@timriley Isn't this a way to protect the writer from trivial mistakes? Why disabling it?

Reply:
Seems like @solnic added this one fairly recently: https://github.com/dry-rb/devtools/commit/74bd1f6f30c7f7cfaa425f87faf65a703910f050

I'm not sure why it's there, I think we can remove it.